Factorial/src/Factorial.java

```diff
@@ -0,0 +1,80 @@
+import org.junit.jupiter.api.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+import static org.junit.jupiter.api.Assertions.*;
+
+/* Reads an integer and prints n! = 1 * 2 * ... * n to standard output
+   For eg: 0! = 1, 1! = 1, 2! = 2 * 1 = 2, 3! = 3 * 2 * 1 = 6 ..  so on
```

Comment:
Why comments? Your unit test cases are documentation enough

Reply:
Will remove them, had to post that on IG page.
Thanks.